lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -182,16 +182,10 @@
       - Setting this will force container to be restarted.
   log_driver:
     description:
-      - Specify the logging driver. Docker uses json-file by default.
-    choices:
-      - none
-      - json-file
-      - syslog
-      - journald
-      - gelf
-      - fluentd
-      - awslogs
-      - splunk
+      - Specify the logging driver. Docker uses I(json-file) by default.
+      - See L(here,https://docs.docker.com/config/containers/logging/configure/) for possible choices.
+    default: null
```

Comment:
I think it's better to not specify `default` if there isn't one. Otherwise, something strange might show up in the automatically generated documentation :)